### PR TITLE
[pfcwd]: Log PFC frame for a lossy queue

### PR DESCRIPTION
**Pull request by marian-pritsak**
Signed-off-by: marian-pritsak <marianp@mellanox.com>

### CODE REVIEW

**Comment by lguohan** [comment]
@sihuihan88 to review

File: orchagent/pfcwdorch.cpp [hunk 1]
```diff
@@ -31,6 +31,99 @@ PfcWdOrch<DropHandler, ForwardHandler>::PfcWdOrch(DBConnector *db, vector<string
...
+        const auto field = fvField(fv);
+        const auto value = fvValue(fv);
+
+
+        for (size_t prio = 0; prio != counterNames.size(); prio++)
+        {
+            if (field == counterNames[prio])
+            {
+                counters[prio] = stoul(value);
+            }
```
Comment (sihuihan88) [inline orchagent/pfcwdorch.cpp]: Can we log warning if SAI_PORT_STAT_PFC_[n]_RX_PKTS is not retrieved for any priority?

File: orchagent/pfcwdorch.cpp [hunk 2]
```diff
@@ -31,6 +31,99 @@ PfcWdOrch<DropHandler, ForwardHandler>::PfcWdOrch(DBConnector *db, vector<string
...
+        }
+
+        uint8_t pfcMask = attr.value.u8;
+
+        for (size_t prio = 0; prio != counters.size(); prio++)
+        {
+            bool isLossy = ((1 << prio) & pfcMask) == 0;
+            if (isLossy && counters[prio] < newCounters[prio])
+            {
+                SWSS_LOG_WARN("Got PFC frame on lossy queue %lu port %s", prio, port.m_alias.c_str());
```
Comment (sihuihan88) [inline orchagent/pfcwdorch.cpp]: Can we also log the counter value?

File: orchagent/pfcwdorch.cpp (hunk 1 above)
Comment (marian-pritsak) [reply orchagent/pfcwdorch.cpp]: Done

File: orchagent/pfcwdorch.cpp (hunk 2 above)
Comment (marian-pritsak) [reply orchagent/pfcwdorch.cpp]: Done

**Pull request merged**